src/c++/vdo/fake/linux/highmem.h

```diff
@@ -1,8 +1,8 @@
-/* SPDX-License-Identifier: GPL-2.0-only */
+/* SPDX-License-Identifier: GPL-2.0 */
 /*
```

Comment:
Must other files have license GPL-2.0-only. I think removing the "-only" here was a mistake? (Not that licensing on these matters all that much...)